docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+# Tizen 5.0 Public M2 Release Notes
+
+Release date: Oct. 31, 2018
+
+
+## System (Kernel and System framework)
+
+### New and changed features
+
+- New API set and features
+  - D-Bus
+    - A policy rule checker has been developed and deployed for improving security.
+  - Dlog
+    - A new feature of front-end filter has been developed for efficient logging and facilitating runtime log analysis.
+  - Deviced
+    - C# API set for display control has been added.
+    - Native API set for multi-LED control has been added.
+  - Storaged
+    - Native and C# API set for storage device types has been added.
+    - A new storage cleanup policy has been developed.
+  - Compressed fs
+    - Btrfs, Squashfs, and Overlayfs have been supported for rootfs.
+- Refactoring and Maintenance
+  - Systemd/Multi-user
+    - Lazy mount units and API set has been added for improving developer experience.
+    - Passwd files have been divided into read-only and read-write files for better reliability and security.
+    - Hashing for the root password has been reinforced from MD5 to SHA-512.
+  - Open source Upgrade
+    - Squashfs has been upgraded to the version of 4.3.
+    - Btrfs-progs has been upgraded to the version of 4.16.1.
+- Testsuite
+  - Deviced HAL TC
+    - A new testsuite for Deviced HAL API set has been developed.
+  - D-Bus
+    - A new testsuite for D-Bus policy modules has been developed.
+  - Dlog
+    - A new testsuite including unit and stress tests has been developed.
+
+### Fixes
+
+- Systemd 
+  - CVE-2018-1049 has been patched. 
+- D-Bus
+  - Duplicated and unnecessary D-Bus policy rules have been removed.
+  - dbus-glib (deprecated library) has been removed.
+- USB-Host
+  - Race condition problems in TCT and libraries have been fixed.
+
+
+## System (IoT System)
+
+### New and changed features
+
+- Debugging tools have been enhanced to obtain stability and reliability quickly during development, the tools help to reduce product development period for bringing up new devices:
+  - Mini Coredump has been provided by reducing unnecessary sections from default coredump for transmitting through network and for reducing truncated coredumps by sudden process cleanup.
+  - Linux-based callstack symbol resolution tools have been provided to integrate into CI/CD infrastructure.

```

Comment:
Is there a specific reason to hyphenate the following words:
- Linux-based
- Thread-based

Reply:
No.

Reply:
@Reni-90,
As s-baik commented, if there's no specific reason, what would you sugget?

Reply:
Then, please rephrase as:
- Linux based
- Thread based

Reply:
will accept